.gitignore

```diff
@@ -9,6 +9,9 @@
 # production
 /build
 
+# editor stuff
+/.vscode
```

Comment:
I think we might want to include this folder because it could help anyone who uses vscode to get moving faster.

Reply:
I personally leave this behind, since not everybody has the same preferences per editor, plus this folder gets updated if you make changes on your workspace which can cause conflicts or undesired changes on other people dev setup.

Reply:
Ok, consider me swayed. :)